app/controllers/platform_admin/photos_controller.rb

```diff
@@ -3,10 +3,33 @@ def index
     @photos = Photo.all
   end
 
+  def edit
+    @photo = Photo.find(params[:id])
+  end
+
+  def update
+    photo = Photo.find(params[:id])
+    # photo.update(photo_params)
+    vendor = Vendor.find(photo.vendor.id)
+    vendor.photos.find(photo.id).update(photo_params)
```

Comment:
I think if we're not updating the vendor here we should be able to just directly update the photo. What's wrong with the commented out line 12? Think we need vendor.photos.create when we create the photo to establish the relationship, but now that we have that, should be able to directly edit the photo. Recommend using line 12 and deleting 13 and 14. If there's a reason not to that I'm missing, recommend deleting line 12.